fixtures/packaging/babel-standalone/dev.html

```diff
@@ -1,12 +1,12 @@
 <html>
   <body>
-    <script src="../../../build/dist/react.development.js"></script>
-    <script src="../../../build/dist/react-dom.development.js"></script>
+    <script src="../../attribute-behavior/public/react.development.js"></script>
+    <script src="../../attribute-behavior/public/react-dom.development.js"></script>
     <script src="https://unpkg.com/babel-standalone@6/babel.js"></script>
     <div id="container"></div>
     <script type="text/babel">
       ReactDOM.render(
-        <h1>Hello World!</h1>,
+        <svg><hkern K={'lo;'} /></svg>,
```

Comment:
We probably don't intend to merge this part, right?

Reply:
Nope :P

Reply:
Lol. I honestly thought that was an edge case test. 😨